docker/prod/Dockerfile

```diff
@@ -169,9 +169,15 @@ COPY vendor ./vendor
 # some gemspec files of plugins require files in there, notably OpenProject::Version
 COPY lib ./lib
 
-RUN bundle install --quiet --deployment --path vendor/bundle --no-cache \
-  --with="$RAILS_GROUPS" --without="test development" --jobs=8 --retry=3 && \
-  rm -rf vendor/bundle/ruby/*/cache && rm -rf vendor/bundle/ruby/*/gems/*/spec && rm -rf vendor/bundle/ruby/*/gems/*/test
+RUN \
+  bundle config set --local path 'vendor/bundle' && \
+  bundle config set --local with '$RAILS_GROUPS' && \
```

Comment:
Shouldn't it be double quotes?
This would set the following in `.bundle/config`:
```
BUNDLE_WITH: "$RAILS_GROUPS"
```
But I could not figure out where this `RAILS_GROUPS` is set.

Reply:
`RAILS_GROUPS` was set to `production` before 997dd6eaeb where this variable was removed.